docs/dml-of-carbondata.md

```diff
@@ -316,12 +316,14 @@ CarbonData DML statements are documented here,which includes:
   You can use this command to insert them into the table, so that making them visible for query.
   
   ```
-  INSERT INTO <CARBONDATA TABLE> STAGE
+  INSERT INTO <CARBONDATA TABLE> STAGE OPTIONS(property_name=property_value, ...)
```

Comment:
please describe all supported options in this document

Reply:
done